validator_derive/tests/contains.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(deprecated)]
```

Comment:
why do we need that in many test files?

Reply:
I mentioned the other day that _"all tests currently on the master branch would pass without modification"_. This is true but, because the `inner` function is now marked as deprecated, the test run logs warnings every time it is used.
I added the `allow` tags as I didn't know if Travis would fail the build with warnings but am happy to remove them if this is not the case.
The alternative is to update the tests to use the new `field_errors` method instead.
Any preference?

Reply:
I'm tempted to just release a new major version so we can probably get rid of `inner` completely

Reply:
Your call :)

Reply:
Perhaps I'll update the test to use the new method and strip out the allow tags anyway.
Then you can decide what version to release either way.